README.md

```diff
@@ -20,77 +20,57 @@
+VDK is built for resiliency and can recover in mid-process or restart entirely from the start.
 
 
-#### Data Journey and where VDK fits in
+# Data Journey and Versatile Data Kit
+VDK creates data processing workflows to:
+- Ingest data (extract)
+- Transform data (transform)
+- Export data (load)
 ![Data Journey](./support/images/versatile-data-kit-data-journey.svg#gh-light-mode-only)
 ![Data Journey](./support/images/versatile-data-kit-data-journey-dark-mode.svg#gh-dark-mode-only)
 
-# Installation and Getting Started
-
-#### Install Versatile Data Kit SDK
-
-```bash
-pip install -U pip setuptools wheel
-pip install quickstart-vdk
-```
-Note that Versatile Data Kit requires Python 3.7+.
 
-See the [Installation page](https://github.com/vmware/versatile-data-kit/wiki/Installation#install-sdk) for more details.
+# Solve common data engineering problems
+- Ingest data from different sources, including CSV files, JSON objects, and data from REST API services.
+- Ensure data applications are packaged, versioned, and deployed correctly while dealing with credentials, retries, and reconnects.
+- Provide built-in monitoring and smart notification capabilities.
+- Track both code and data modifications and the relationship between them, allowing quicker troubleshooting and version rollback.
 
-#### Use
+![Without / With Versatile Data Kit](./support/images/versatile-data-kit-before-after-light.svg#gh-light-mode-only)
+![Without / With Versatile Data Kit](./support/images/versatile-data-kit-before-after-dark.svg#gh-dark-mode-only)
 
-```bash
-# see Help to see what you can do
-vdk --help
-```
-Check out the [Getting Started page](https://github.com/vmware/versatile-data-kit/wiki/getting-started) to create and run your first Data Job.
+# Versatile Data Kit Components
+- Software Development Kit (SDK):
+    - Tools to automate the extraction, transformation, and loading of data.
+    - A plugin framework that allows users to extend the framework according to their specific requirements.
+- Control Service: The Control Service allows users to create, deploy, manage, and execute data jobs in a Kubernetes runtime environment.
 
-# Documentation
+# Getting Started
+Installing VDK is a simple pip command. See the [Getting Started](https://github.com/vmware/versatile-data-kit/wiki/getting-started) guide to install VDK and create a data job.
 
-Official documentation for Versatile Data Kit can be found [here](https://github.com/vmware/versatile-data-kit/wiki/Introduction).
+# Next Steps
+- See [use case examples](https://github.com/vmware/versatile-data-kit/wiki/Examples) that show how VDK fits into the data workflow.
+- See the [documentation](https://github.com/vmware/versatile-data-kit/wiki/Introduction) for VDK.
+- See the article about [using the Versatile Data Kit and Trino DB](https://towardsdatascience.com/from-raw-data-to-a-cleaned-database-a-deep-dive-into-versatile-data-kit-ab5fd992a02e).
```

Comment:
Should we mention somewhere our community resources - https://github.com/vmware/versatile-data-kit/wiki/Community-and-Resources

Reply:
updated